CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Implementation of the RabbitMQ EventBus
 - DeadLetter APIs and memory implementation for storing events that failed delivery
 - RecipientRewriteTable Aliases and associated WebAdmin routes
+- Dead letter reDeliver routes on top of WebAdmin
+- Dead letter Cassandra implementation
+- Deleted Message vault implementation on top of MailReposiotries
```

Comment:
Do we want to keep that implementation? I thought it was a transient state. If so, why should we document it?